biogrid/libs/grid-simulator/src/lib/state/state-interface.ts

```diff
@@ -20,6 +20,7 @@ export interface StateGraph {
   getAllPositions: () => ItemPosition[];
   getGridItem: (name: string) => GridItem;
   getGraph: () => Graph;
+  getJsonGraph: () => any;
```

Comment:
this is not implemented anywhere? could you remove it or is it implemented?

Reply:
It is in biogrid state and biogrid